File: Hoops.Core/Entities/Household.cs
using System;
using System.Collections.Generic;

namespace Hoops.Core.Entities
{
    public partial class Household
    {
        public int HouseId { get; set; }
        public int? CompanyId { get; set; }
        public string Name { get; set; }
        public string Phone { get; set; }
        public string Address1 { get; set; }
        public string Address2 { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Zip { get; set; }
        public string Email { get; set; }
        public bool? EmailList { get; set; }
        public string SportsCard { get; set; }
        public int? Guardian { get; set; }
        public bool? FeeWaived { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string CreatedUser { get; set; }
        public int? Teamid { get; set; }
    }
}

File: hoops.ui/src/app/domain/household.ts
export class Household {
  public houseId: number = 0;
  public name!: string;
  public address1?: string;
  public address2?: string;
  public city?: string;
  public state?: string = 'FL';
  public zip?: string;
  public email?: string;
  public phone?: string;
}

// Interface for household list display
export interface HouseholdListItem {
  id: number; // For BaseList compatibility
  houseId: number;
  name: string;
  address1?: string;
  phone?: string;
  email?: string;
}
